src/Swarrot/Processor/ConfigurableInterface.php

```diff
@@ -6,8 +6,5 @@
 
 interface ConfigurableInterface extends ProcessorInterface
 {
-    /**
-     * setDefaultOptions.
```

Comment:
We should probably add `@return void` in the phpdoc of 3.7, so that Symfony's DebugClassLoader can warn about the return type to prepare the migration.

Reply:
Fixed in #214